### PR TITLE
Add code in Dockerfile.migraphx-ci to install the onnxruntime for accuracy_checker. 

### PR DESCRIPTION
**Pull request by stefankoncarevic**

### CODE REVIEW

File: mlir/utils/jenkins/Jenkinsfile [hunk 1]
```diff
@@ -878,6 +879,11 @@ pipeline {
                                     }
                                 }
                             }
+                            //Accuracy_checker will compare outputs from MIGraphX and onnx runtime
+                            dir('MIGraphX/tools/accuracy') {
+                                sh 'pip3 install -r requirements.txt'
```
Comment (krzysz00) [inline mlir/utils/jenkins/Jenkinsfile]: I'm not happy with this CI-time install.

Could you move the requirements into the relevant Dockerfile instead?

**Review by krzysz00 — CHANGES_REQUESTED** [review]

File: mlir/utils/jenkins/Jenkinsfile (hunk 1 above)
Comment (stefankoncarevic) [reply mlir/utils/jenkins/Jenkinsfile]: Thank you, I will remove that from Jenkinsfile and add to Dockerfile.

**Comment by stefankoncarevic** [comment]
@krzysz00 
After your previous comment for CI-time install, I open the requirements and see the only two things to be install, one is the numpy and other is the onnxruntime. Because we have numpy already install, I add in our Dockerfile.migraphx-ci:
 RUN pip3 install onnxruntime==1.10.0.  After finish CI, I saw the message from MIGraphX: 
ModuleNotFoundError: No module named 'onnxruntime'.
I think that message is because onnxruntime is not install from Dockerfile.migraphx-ci? 
Is it a good way and place, where I wrote, or should it be different?

**Comment by krzysz00** [comment]
@stefankoncarevic You'll need to make a separate commit for the Dockerfile changes

**Comment by stefankoncarevic** [comment]
Thank you.

**Review by krzysz00 — APPROVED** [review]

**Pull request merged**